providers/shared/attributesets/attributeset.ftl

```diff
@@ -0,0 +1,3 @@
+[#ftl]
+
+[#assign LINK_ATTRIBUTESET_TYPE = "Link"]
```

Comment:
Maybe make lower case in line with things like component types

Reply:
@ml019 updated in https://github.com/hamlet-io/engine/pull/1498/commits/835a9271a905ef6794e7aec78907ef492cf2f157